Mathlib/Analysis/InnerProductSpace/PiL2.lean

```diff
@@ -1028,6 +1028,21 @@
   rfl
 #align matrix.pi_Lp_equiv_to_euclidean_lin Matrix.piLp_equiv_toEuclideanLin
 
+theorem toEuclideanLin_apply (M : Matrix m n 𝕜) (v : EuclideanSpace 𝕜 n) :
+    toEuclideanLin M v =
+      (WithLp.equiv 2 (m → 𝕜)).symm (M *ᵥ (WithLp.equiv 2 (n → 𝕜)) v) :=
+  rfl
+
+@[simp]
+theorem Matrix.piLp_equiv_symm_toEuclideanLin_apply (M : Matrix m n 𝕜) (v : EuclideanSpace 𝕜 n) :
+   WithLp.equiv 2 (m → 𝕜) (toEuclideanLin M v) = (M *ᵥ (WithLp.equiv 2 (n → 𝕜)) v) :=
```

Comment:
Since I butchered these changes, Jireh had me revert the state to before the above suggestions were made, and provided names for the lemmas. (Thanks again, man!!!)

Reply:
This isn't your fault, this is a bug in the bot

Reply:
Whew! That's a relief. Thanks, Eric!

Reply:
In fact, I think every one of these bot comments is garbage; it should only be suggesting whitespace changes.